test/test_gradient.cpp

```diff
@@ -0,0 +1,269 @@
+/* A utility to transfer tps solutions from one mesh to another */
```

Comment:
This looks like the commented out test.  Is it ready to be included?  If so, we should do that.  If not, maybe this should part of a separate PR.

Reply:
Gradient test is now added in the ci test.